fendermint/testing/materializer/src/logging.rs

```diff
@@ -40,13 +40,13 @@ where
     M::Relayer: Display,
 {
     async fn create_network(&mut self, testnet_name: &TestnetName) -> anyhow::Result<M::Network> {
-        eprintln!("create_network({testnet_name:?}");
+        // eprintln!("create_network({testnet_name:?}");
```

Comment:
Why not delete the commented code in this file, we should log to console with the tracing crate

Reply:
I can delete it, no problem. Sometimes I find it easier to just add these `eprintln!` logs to test to see what's happening, rather than enable the logs on tests which can bring in more noise, and just generally takes much longer to figure out how to enable the way I want. But fair enough, no need to keep these.